### PR TITLE
fixes #12000 - support remote execution in installer

### PR DESCRIPTION
**Pull request by stbenjam**
Still a WIP, need to figure out how to handle the foreman proxy plugins, as foreman proxy isn't at the top level.

### CODE REVIEW

File: config/katello-installer.yaml [hunk 1]
```diff
@@ -45,6 +45,9 @@
     :foreman::plugin::templates:
       :dir_name: foreman
       :manifest_name: plugin/templates
+    :foreman::plugin::remote_execution:
+      :dir_name: foreman
+      :manifest_name: plugin/remote_execution
```
Comment (ekohl) [inline config/katello-installer.yaml]: You can add ```:params_name: plugin/remote_execution/params``` (see https://github.com/theforeman/foreman-installer/blob/42521b79e4e4a7a05dbf30b6fd5a2b783fb2e871/config/foreman-installer.yaml#L133-L136 for an example)

Comment (stbenjam) [inline config/katello-installer.yaml]: Is it required? Not all plugins have that defined

Comment (ekohl) [inline config/katello-installer.yaml]: Not all plugins have params. This allows kafo to load the defaults from the params file if it's used.

**Comment by stbenjam** [comment]
Capsule PR is https://github.com/Katello/puppet-capsule/pull/62

File: config/katello-installer.yaml (hunk 1 above)
Comment (stbenjam) [inline config/katello-installer.yaml]: added

**Comment by ekohl** [comment]
Untested, but I think it looks good.

**Comment by stbenjam** [comment]
@katello/installer Test mention please ignore

**Comment by stbenjam** [comment]
@Katello/installer hello

**Comment by ehelms** [comment]
Is this still waiting on you (other than needing a rebase) ? Did you want to get this into 2.4?

**Comment by stbenjam** [comment]
I think it was good to go, just needed a rebase

**Comment by ehelms** [comment]
ACK

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**